x-pack/packages/kbn-elastic-assistant/impl/connectorland/connector_missing_callout/index.tsx

```diff
@@ -37,27 +38,33 @@ export const ConnectorMissingCallout: React.FC<Props> = React.memo(
     }, [isSettingsModalVisible, setIsSettingsModalVisible, setSelectedSettingsTab]);
 
     return (
-      <EuiCallOut
-        color="danger"
-        iconType="controlsVertical"
-        size="m"
-        title={i18n.MISSING_CONNECTOR_CALLOUT_TITLE}
-      >
-        <p>
-          {' '}
-          <FormattedMessage
-            defaultMessage="Select a connector above or from the {link} to continue"
-            id="xpack.elasticAssistant.assistant.connectors.connectorMissingCallout.calloutDescription"
-            values={{
-              link: (
-                <EuiLink onClick={onConversationSettingsClicked}>
-                  {i18n.MISSING_CONNECTOR_CONVERSATION_SETTINGS_LINK}
-                </EuiLink>
-              ),
-            }}
-          />
-        </p>
-      </EuiCallOut>
+      <>
+        {assistantAvailability.hasConnectorsAllPrivilege ? (
```

Comment:
Found bug in testing here, need additional check if READ && at least one connector is configured so we don't show the 'privilege required' button'. Flow isn't actually broken, just UI shown. Adding test for this case as well 👍

Reply:
Bug fixed and tests added in [020ecec](https://github.com/elastic/kibana/pull/164382/commits/020ecec1649a83d653422d22c06b5a6646189703).